cordial_face/scripts/client_to_start_face_on_pi.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
In general, I think it would be nice to separate out the QT-specific scripts from the general purpose ones